src/components/Disputes/DisputeTimeline.js

```diff
@@ -164,33 +182,116 @@ function ItemStep({ item, index, roundStepContainer }) {
   )
 }
 
+function Outcome({ outcome, phase }) {
+  const theme = useTheme()
+  const title =
+    phase && phase === DisputePhase.RevealVote ? 'JURY OUTCOME' : 'OUTCOME'
+
+  return (
+    <React.Fragment>
+      <div
+        css={`
+          margin-top: ${1 * GU}px;
+        `}
+      >
+        <span
+          css={`
+            ${textStyle('body3')}
+            color:${theme.contentSecondary};
+          `}
+        >
+          {title}
+        </span>
+      </div>
+      <OutcomeText outcome={outcome} phase={phase} />
+    </React.Fragment>
+  )
+}
+
+function OutcomeText({ outcome, phase }) {
+  const theme = useTheme()
+
+  const { Icon, color } = getOutcomeIcon(outcome, theme)
+
+  let outcomeText
+  if (phase === DisputePhase.RevealVote) {
+    outcomeText = juryOutcomeToString(outcome)
+  } else {
+    const confirm = phase === DisputePhase.ConfirmAppeal
+    outcomeText = appealRulingToString(outcome, confirm)
+  }
+
+  return (
+    <div>
+      {Icon && (
+        <div
+          css={`
+            color: ${color};
+            display: flex;
+            align-items: center;
+          `}
+        >
+          <Icon size="medium" />
+          <span
+            css={`
+              ${textStyle('body2')}
+            `}
+          >
+            {outcomeText}
+          </span>
+        </div>
+      )}
+    </div>
+  )
+}
+
+export function getOutcomeIcon(outcome, theme) {
```

Comment:
Should we make this a hook rather than passing `theme`, and call it something like `useOutcomeStyle()` since it also returns a color?

Reply:
Yes, good idea